DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -4,8 +4,8 @@
 
 @{
   var application = (string)ViewData["Application"] ?? "";
-  var shouldDisplayLogoutMenuItem = false;
   var shouldDisplayHelpMenuItem = application != "Tracking System";
+  var headerApplicationName = application == "Main" ? "" : application;

```

Comment:
Do we ever use `ApplicationType.ApplicationName` for anything other than this? If not, let's change the name of the property to something like `HeaderExtension`.